managed/services/management/common/common.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package common contains common and cross-service logics.
+package common
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"gopkg.in/reform.v1"
+
+	backuppb "github.com/percona/pmm/api/managementpb/backup"
+	"github.com/percona/pmm/managed/models"
+	managementbackup "github.com/percona/pmm/managed/services/management/backup"
+)
+
+// ErrClusterBlocked is returned when there is an unfinished job that doesn't allow to change service cluster name.
+var ErrClusterBlocked = errors.New("cluster/service is blocked")
```

Comment:
Shouldn't it be `Locked` vs `Blocked`? IMO it better reflects our intent to temporarily disallow certain changes.